pkg/ecdsa/dkg/dkg.go

```diff
@@ -23,24 +54,26 @@ func Execute(
 	channel net.BroadcastChannel,
 	membershipValidator *group.MembershipValidator,
 ) (*Result, uint64, error) {
-	logger.Debugf("[member:%v] initializing member", memberIndex)
+	e.logger.Debugf("[member:%v] initializing member", memberIndex)
 
 	registerUnmarshallers(channel)
 
 	member := newMember(
-		logger,
+		e.logger,
 		memberIndex,
 		groupSize,
 		dishonestThreshold,
 		membershipValidator,
+		seed.Text(16), // TODO: Should change on retry.,
+		e.tssPreParamsPool.get(),
```

Comment:
For an early version when we'll not have that many participants, we need to ensure the pool size is... large.

Reply:
This is true!